api/specs/common/schemas/scripts/remove_definitions.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+import yaml
+
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+source_file_name = sys.argv[1]
+target_file_name = sys.argv[2]
+file_source_path = DIR_PATH + f"/../{source_file_name}"
+file_target_path = DIR_PATH + f"/../{target_file_name}"
```

Comment:
minor: you could use argparse to make it more clean and at that point you could use a reference to the fine from the file system instead of composing it.

Reply:
Forgot this comment?

Reply:
Sorry missed this comment, thanks for the tip! I will use it next time. This is just a small helper script, that will be removed in the future with additional refactoring. This was just the first step to making project and node pydantic models json schemas for validation. In the future, we would like to remove even this json schema validation and use pydantiv model straight away.